CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - **Breaking change** The api now uses the module name instead of the prefix to generate the storage methods. The methods of the grandpa module changed from `api.query.grandpaFinality` to `api.query.grandpa`. 
 - Update with latest substrate 2.x types
 - **Breaking Change** StorageFunction has been renamed to StorageEntry.
+- **Breaking Change** Vote interface extends U8a instead of I8.
```

Comment:
```suggestion
- **Breaking Change** Vote interface extends U8a instead of I8 and no longer supports the `toNumber()` method. Vote properties can be accessed via the `isAye`, `isNay`, and `conviction` getters. 
Votes can still be constructed as before with a raw JS boolean, a SCALE encoded Boolean, an i8 number, or a JS object with properties `aye` and `conviction` defined. 
```

Reply:
```suggestion
- **Breaking Change** Vote interface extends U8a instead of I8. Vote properties can be accessed via the `isAye`, `isNay`, and `conviction` getters. 
Votes can still be constructed as before with a raw JS boolean, a SCALE encoded Boolean, an i8 number, or a JS object with properties `aye` and `conviction` defined. 
```

Reply:
I the only thing that would have been breaking would be if people were using `toNumber()`, which it doesn't look like apps or any or the other interfaces were. I will document it better anywhow